openvdb_points/tools/AttributeSet.cc

```diff
@@ -168,6 +180,74 @@ AttributeSet::get(size_t pos)
 }
 
 
+size_t
+AttributeSet::groupOffset(const Name& group) const
+{
+    const Descriptor::NameToPosMap& map = this->descriptor().groupMap();
+    if (map.find(group) == map.end()) {
+        return INVALID_POS;
+    }
+    return map.at(group);
```

Comment:
This page says the 'at' method was introduced in C++11 and may not be supported by older compilers. You may want to avoid a second lookup here by reusing the iterator returned by 'find' above...